### PR TITLE
Add missing "format" facet in Date type

### LINKED ISSUE #728: Add missing "format" facet in Date type
The ["Date" type](https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md#date) definition section mentions the "additional facet `format`" multiple times however it is not formated as a table like in the other sections

### Proposed Change
Reformat the sentence below as a table:
> The additional facet format MUST be available only when the type equals datetime, and the value MUST be either rfc3339 or rfc2616. Any other values are invalid.

E.g.

Facet | Description
------------ | -------------
format? | The additional facet format MUST be available only when the type equals datetime, and the value MUST be either rfc3339 or rfc2616. Any other values are invalid.

### PR DESCRIPTION
**Pull request by jstoiko**
Closes #728

### CODE REVIEW

File: versions/raml-10/raml-10.md [hunk 1]
```diff
@@ -987,7 +987,11 @@ The following date type representations MUST be supported:
 | datetime-only | Combined date-only and time-only with a separator of "T", namely yyyy-mm-ddThh:mm:ss\[.ff...\]. Does not support a time zone offset.
 | datetime | A timestamp in one of the following formats:  if the _format_ is omitted or set to `rfc3339`, uses the "date-time" notation of [RFC3339](http://xml2rfc.ietf.org/public/rfc/html/rfc3339.html#anchor14); if _format_ is set to `rfc2616`, uses the format defined in [RFC2616](https://www.ietf.org/rfc/rfc2616.txt).
 
-The additional facet _format_ MUST be available only when the type equals _datetime_, and the value MUST be either `rfc3339` or `rfc2616`. Any other values are invalid.
+The additional facet `format` MUST be available only when the type equals `datetime`:
```
Comment (dukesphere) [inline versions/raml-10/raml-10.md:990]:
```suggestion
The additional facet `format?` MUST be available only when the type equals `datetime`:
```

Not sure if the "?" is required there.

**Review by dukesphere — CHANGES_REQUESTED** [review]
is the facet called `format?` or `format`?

File: versions/raml-10/raml-10.md (hunk 1 above)
Comment (jstoiko) [reply versions/raml-10/raml-10.md:990]: It's not. The name of the facet is "format", the trailing "?" only indicates that it's optional when listed in a table format as it is below, out of this context the "?" is irrelevant.

**Comment by jstoiko** [comment]
> is the facet called `format?` or `format`?

`format`

**Review by dukesphere — APPROVED** [review]

**Pull request merged**